src/patternfly/components/Table/docs/table-simple.md

```diff
@@ -9,6 +9,7 @@
 
 | Class | Applied to | Outcome |
 | -- | -- | -- |
+| `.pf-c-table-container` | `<div>` | Initiates a table container element. The table container element controls overflow when `.pf-c-table` runs out of space within its container. |
```

Comment:
this doesn't follow any of our other components, since it's a new component. I'm also not sure we need a component for this. If a user's content overflows the table, they could try and ensure that doesn't happen, or if it does, they may need to write CSS, or we could offer overflow options on the elements that wrap the table, like page sections and/or cards. Just some other ideas...